services/QuillLMS/app/models/mailer/user.rb

```diff
@@ -2,6 +2,35 @@
 
 module Mailer
   class User < SimpleDelegator
+
+    def send_school_admin_email(school_id, new_user)
+      school = School.find_by(id: school_id)
+      linked_school = self.school
+
+      return unless school
+
+      if new_user
+        send_internal_tool_admin_account_created_email(school.name) && return
+      end
+
+      if !linked_school
+        send_internal_tool_made_school_admin_link_school_email(school)
+      elsif school == linked_school
+        send_internal_tool_made_school_admin_email(school.name)
+      elsif school != linked_school
+        send_internal_tool_made_school_admin_change_school_email(school, linked_school)
+      end
+    end
+
+    def send_district_admin_email(district_id, new_user)
+      district_name = District.find_by(id: district_id)&.name
+      if new_user
+        send_internal_tool_district_admin_account_created_email(district_name)
+      else
+        send_internal_tool_made_district_admin_email(district_name)
+      end
+    end
```

Comment:
I think elsewhere, we use workers for all the mailers (and it looks like you created them in an earlier PR for this project). I'm just curious about why we aren't using them here. I also think that for the sake of consistency, we might want to move this logic to the controller (mirroring the school admin account flows). It seems like the mailers are pretty logic-lightweight in general.